packages/themes/block-theme/templates/singular.html

```diff
@@ -1,3 +1,7 @@
 <!-- wp:template-part {"slug":"header","area":"header","tagName":"header","className":"header-wrapper"} /-->
-<!-- wp:post-content {"tagName":"main","align":"full","layout":{"type":"constrained"}} /-->
+<!-- wp:group {"tagName":"main","layout":{"type":"constrained"},"className":"content-wrapper"} -->
+<main class="wp-block-group content-wrapper">
```

Comment:
Very unnecessary to wrap Post Content. You should target `.is-layout-flow` instead of `.wp-block-group`/`.wp-block-post-content`. There are a few additional container, like Media & Text that also needs this that already have `is-layout-flow` in place.